lib/apirequest.js

```diff
@@ -184,6 +186,10 @@ function createAPIRequest(parameters, callback) {
   delete options.auth; // is overridden by our auth code
   delete options.params; // We handle params ourselves and Request does not recognise 'params'
 
+  if (missingCallback) {
+    return options;
```

Comment:
Most likely but I'll need to find an API i can play with that uses body before figuring out how to incorporate it